### PR TITLE
Added new translations and few improvement for MP2

### PR DESCRIPTION
**Pull request by SeolYang**
# SCP:SL Translation PR Template

## Language
Korean

## Type of change
- [ ] Added new language translation
- [x] Changed a currently existing translation

## Have you tested the changes you've made?
- [ ] Yes
- [x] No

# Checklist:
- [x] I have made sure to not commit changes to other stuff than my own or the default english one
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x] I have made sure that I'm not submitting a duplicate translation 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name
- [ ] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of new translations

### CODE REVIEW

File: Translations/ko/MenuWarnings.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+경고
+예
+<b>지원하지 않는 Inter graphics</b>가 감지되었 습니다./nlt 는 <b>외장형 그래픽 카드</b>의 사용을 <b>권장</b>하고 있습니다./n 현재 세팅에서 게임이 <b>가끔 멈출수도</b>있습니다.
+음성 채팅을 할 수 없을때는 중요한 시스템 업데이트가 요구될 수도 있습니다. 윈도우를 최신 버전으로 유지하여 주십시오, 그렇지 않으면 게임이 정상적으로 작동하지 않을 수 있습니다. 만약 이런 문제가 지속된다면, 우리의 기술 지원 팀으로 연락하여 추가 지원을 받으시길 바랍니다.
+© 2019 GitHub, Inc.
```
Comment (MichalPetryka) [inline Translations/ko/MenuWarnings.txt]: What's this?

Comment (HoRangDev) [reply Translations/ko/MenuWarnings.txt]: Thanks for reviewing it.
 I copied original text from github pages, so github copyright text also contained together. I didn't expecting that.
 Now that i have corrected the problem, could you review it again?

**Comment by MichalPetryka** [comment]
You've made a typo in Abilities.txt file name.

**Comment by HoRangDev** [comment]
> You've made a typo in Abilities.txt file name.

Done! Also, I checked that applied properly :)

**Comment by ShingekiNoRex** [comment]
Please resolve the conflicts

**Comment by HoRangDev** [comment]
> Please resolve the conflicts
Resolved it!

**Comment by Dat-Joseph** [comment]
Thank you for contributing SeolYang. After all this time we will merge it. (Sorry for the wait!) 😅

**Pull request merged**